courses-app/src/app/app.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { LoginComponent } from './features/login/login.component';
+import { RegistrationComponent } from './features/registration/registration.component';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { HeaderComponent } from './shared/components/header/header.component';
+import { ButtonComponent } from './shared/components/button/button.component';
+import { InfoComponent } from './shared/components/info/info.component';
+import {CoursesModule} from "./features/courses/courses.module";
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    LoginComponent,
+    RegistrationComponent,
+    HeaderComponent,
+    ButtonComponent,
+    InfoComponent,
```

Comment:
Since this is app.module we should declare this in the shared.module (login and registration in the login.module and registration.module accordigly)